locales/de.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Execute binaries from npm packages.\n%s": "Führt Programme aus, die von npm Paketen bereitgestellt werden.\n%s",
+  "Package to be installed.": "Das zu installierende Paket.",
+  "Location of the npm cache.": "Ort des npm Zwischenspeichers.",
+  "Skip installation if a package is missing.": "Überspringe die Installation falls ein Paket fehlt.",
+  "Path to user npmrc.": "Pfad zur Benuzter_innen npmrc.",
```

Comment:
Cool! Thanks for the explanation! I asked a couple of folks I was hanging out with and they weren't sure what it was about so I figured I'd ask.

TIL!